fe/fe-core/src/main/java/com/starrocks/sql/ast/ColumnAssignment.java

```diff
@@ -0,0 +1,28 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.ParseNode;
+
+public class ColumnAssignment implements ParseNode {
+    private final String column;
+    private final Expr expr;
+
+    public ColumnAssignment(String column, Expr expr) {
+        this.column = column;
+        this.expr = expr;
+    }
+
+    public String getColumn() {
+        return column;
+    }
+
+    public Expr getExpr() {
+        return expr;
+    }
+
+    @Override
+    public String toSql() {
```

Comment:
toSql is no longer a required method for the new ParseNode, and will no longer rely on toSql for the "to sql" function in the future. If there is no need for tosql, this method may not be added.

Reply:
ok